source/default_mode/SymWorld.h

```diff
@@ -369,8 +369,16 @@ class SymWorld : public emp::World<Organism>{
    *
    * Purpose: To determine the location of a valid occupied neighboring position.
    */
-  int GetNeighborHost (size_t i) {
-    const emp::vector<size_t> validNeighbors = GetValidNeighborOrgIDs(i);
+  int GetNeighborHost (size_t id) {
+    // Attempt to use GetRandomNeighborPos first, since it's much faster
+    for (int i = 0; i < 3; i++) {
+      emp::WorldPosition neighbor = GetRandomNeighborPos(id);
+      if (neighbor.IsValid() && neighbor.GetIndex() < GetSize() && pop[neighbor.GetIndex()].Raw())
```

Comment:
neighbor.GetIndex() < GetSize() && pop[neighbor.GetIndex()].Raw() can be replaced with IsOccupied(neighbor)